File: tests/language_completion/fixtures/kotlin/control_flow.kt
fun main() {
  var total = 0
  while (total < 5) {
    total = total + 1
  }
  if (total == 5) {
    println("kotlin_control", "ready", total)
  } else {
    println("kotlin_control", "bad", total)
  }
}

File: tests/language_completion/fixtures/go/control_flow.go
package main

import "fmt"

func main() {
  total := 0
  for total < 5 {
    total = total + 1
  }
  if total == 5 {
    fmt.Println("go_control", "ready", total)
  } else {
    fmt.Println("go_control", "bad", total)
  }
}
